packages/gestalt/src/Row.js

```diff
@@ -39,37 +42,24 @@ type Props = {|
   wrap?: boolean,
 |};
 
-export default function Row({
-  alignItems = 'center',
-  children,
-  gap = 0,
-  height,
-  justifyContent = 'start',
-  width,
-  ...rest
-}: Props): Node {
+// Defaults for align-items and justify-content set in base styles
+export default function Row({ children, gap = 0, ...rest }: Props): Node {
+  const hasGap = gap > 0;
+
+  const baseClasses = classnames(styles.row, {
+    [`horizontalGap${gap}`]: hasGap,
+    [`horizontalGap${gap} > *`]: hasGap,
```

Comment:
@dangerismycat can we remove this line? Otherwise we see the following:

```
className="row horizontalGap4 horizontalGap4 > *"
```

So `horizontalGap4 > *` should get removed